### PR TITLE
Add translation

### PR DESCRIPTION
**Pull request by Simon25500**
Thank you for submitting a pull request!

## Description

Add translation as Maplibregl-export
fix Navigation type errors (maybe due to my version of ts)

## Type of Pull Request

- [x] Adding a feature
- [x] Fixing a bug
- [x] Maintaining documents
- [ ] Others ()

## Verify the followings

- [x] Code is up-to-date with the `main` branch
- [x] No lint errors after `npm run lint`
- [ ] Make sure all the exsiting features working well

Refer to [CONTRIBUTING.MD](https://github.com/watergis/mapbox-gl-export/tree/master/.github/CONTRIBUTING.md) for more details.

### CODE REVIEW

File: lib/map-generator.ts [hunk 1]
```diff
@@ -68,6 +68,14 @@ export const Size = {
...
 type Size = typeof Size[keyof typeof Size];
 
+declare global {
+  // eslint-disable-next-line no-unused-vars
+  interface Navigator {
+    // eslint-disable-next-line no-unused-vars
+    msSaveBlob?: (blob: any, defaultName?: string) => boolean
+  }
+}
+
```
Comment (JinIgarashi) [inline lib/map-generator.ts:78]: Why did you add this?

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
@Simon25500 Thank you for pull request. Could you fix the code because CI failed.

File: lib/map-generator.ts [hunk 2]
```diff
@@ -244,7 +252,7 @@ export default class MapGenerator {
   private toJPEG(canvas: HTMLCanvasElement, fileName: string) {
     const uri = canvas.toDataURL('image/jpeg', 0.85);
     // @ts-ignore
-    if (canvas.msToBlob) {
+    if (canvas.msToBlob && window.navigator?.msSaveBlob) {
```
[the remark this answers is not in the document]
Comment (JinIgarashi) [reply lib/map-generator.ts:255]: @Simon25500 To be honest, I just confirmed my typescript also shows error at `window.navigator.msSaveBlob(blob, fileName);`...

I think we can simply remove this code for IE11 since IE11 is no longer supported officially by Microsoft. What do you think?

**Comment by Simon25500** [comment]
I add this code cause ts wasn't compile i don't know if it's because of my ts version i'll remove that :)

File: lib/map-generator.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Simon25500) [reply lib/map-generator.ts]: @JinIgarashi ah great, I though I was the only one to have this errors ;) but yes I think we can do that !

[the remark this answers is not in the document]
Comment (Simon25500) [reply lib/map-generator.ts]: @JinIgarashi I made the modification

**Review by JinIgarashi — APPROVED** [review]

**Comment by JinIgarashi** [comment]
@Simon25500 Thank you so much! Now I have released it as [v1.2.15](https://github.com/watergis/mapbox-gl-export/releases/tag/v1.2.15)!
